src/Decimal.php

```diff
@@ -831,7 +831,7 @@ protected function fromScientific(string $value, ?int $scale): void
         } else {
             $this->integralPart = bcmul($matches[2], bcpow('10', (string)$exp));
 
-            $pos = strlen((string)$this->integralPart);
+            $pos = strlen($this->integralPart);
```

Comment:
If we remove the string cast here, shouldnt we also remove it in line 841?
https://github.com/spryker/decimal-object/blob/b9c5d2192d6cb2692d5dc1dd63d3d3e4c77d566d/src/Decimal.php#L841